app/controllers/application_controller.rb

```diff
@@ -1,4 +1,5 @@
 require 'google/apis/drive_v2'
+Dir["charts/*.rb"].each {|file| require file }
```

Comment:
I decided my models folder was getting ridiculous, so moved out all the charts related classes into a charts folder.  Is there a better way to get that file seen by Rails than this line?